sentry-core/src/main/java/io/sentry/core/SentryOptions.java

```diff
@@ -228,6 +231,14 @@ public void addInAppInclude(@NotNull String include) {
     inAppIncludes.add(include);
   }
 
+  public @NotNull ITransport getTransport() {
+    return transport;
```

Comment:
Callers would need to be aware of `NoOp` as a type if they'd want to see what's assigned here (can't infer `null` means nothing was set yet)

Reply:
yeah, it makes sense.